packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.ts

```diff
@@ -128,6 +128,8 @@ const getAllProps = (
   return {
     display: getResponsiveValue(props.display, size),
     overflow: getResponsiveValue(props.overflow, size),
+    overflowX: getResponsiveValue(props.overflowX, size),
+    overflowY: getResponsiveValue(props.overflowY, size),
```

Comment:
missed this earlier. If we support `overflow`, we should support `overflowX`, `overflowY` as well.